frontend/src/components/Forms/variances/VarianceSearchForm.js

```diff
@@ -20,7 +20,7 @@ const defaultProps = {
   isAdvanceSearch: false,
 };
 
-const validate = (values) => {
+export const validate = (values) => {
   const errors = {};
```

Comment:
Let's avoid exporting methods in this way, If two files are using the same validate function, move it to a common resource and import it into both files.